go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	k8s.io/client-go v0.16.9
 	k8s.io/code-generator v0.16.9
 	k8s.io/kube-openapi v0.0.0-20190816220812-743ec37842bf
-	volcano.sh/volcano v0.4.0
+	volcano.sh/apis v1.16.15
```

Comment:
l think we also need to update the gomod dependency?

Reply:
Yes, so i close the pr and submit a new one

Reply:
Why not updating this PR directly?

Reply:
Is there any buttons or labels available to restart the CI test?

Reply:
When you push a new commit it will trigger new builds.

Reply:
OK, Thanks.